docs/containers/docker-compose.md

```diff
@@ -0,0 +1,217 @@
+
+To start debugging a running app or service, first make sure the **Attach** configuration you created is selected in the **Debug** tab configuration dropdown, then launch the debugger in the usual way. From the **Debug** tab, choose the green arrow (**Start** button) or use `kb(workbench.action.debug.start)`.
+
+![Screenshot of starting debugging](images/compose/docker-compose-attach.png)
+
+When you attach, the web browser doesn't open automatically, so you need to start the browser on the host and navigate to the app at `http://localhost:{port}`, where the `port` is found by inspecting the .json configuration for the running container as described previously.  
+
+![Screenshot of debug session](./images/compose/docker-compose-debugging.png)
+
+The process for configuring debugging is the same for each platform. For .NET Core, the extension generates the launch configuration for attaching to your service. For Python and Node.js, you can copy and modify the following examples. Copy the code into `launch.json` and modify the settings as needed.
+
+### Node.js
+
+Here's an example that shows the Node.js launch configuration - Attach:
+
+```json
+    "configurations": [
+        {
+            "type": "node",
+            "request": "attach",
+            "name": "Docker: Attach to Node",
+            "remoteRoot": "/usr/src/app",
+            "port": 9229 // This is optional. It will be inferred from the docker-compose.debug.yml file.
+        },
+        // ...
+    ]
+```
+
+### Python
+
+Python doesn't have a *docker-compose.debug.yml*. Here's an example showing the Python launch configuration - Django Attach:
+
+```json
+    "configurations": [
+        {
+            "name": "Docker: Python - Django",
+            "type": "docker",
+            "request": "launch",
+            "preLaunchTask": "docker-run: debug",
+            "python": {
+                "pathMappings": [
+                    {
+                        "localRoot": "${workspaceFolder}",
+                        "remoteRoot": "/app"
+                    }
+                ],
+                "projectType": "django"
+            },
+            // ...
+        }
+    ]
+```
+
+### .NET
+
+An example launch configuration for an ASP.NET web app in a Docker container is shown in the following code:
+
+```json
+        {
+            "name": "Docker .NET Core Attach (Preview)",
+            "type": "docker",
+            "request": "attach",
+            "platform": "netCore",
+            "sourceFileMap": {
+                "/src": "${workspaceFolder}"
+        }
+```
+
+If you try to attach to a .NET Core app running in a container, you'll see a prompt asking if you want to install the debugger (`.vsdbg` bits into the container).
+
+![Screenshot of debugger install prompt](./images/compose/docker-compose-netcore-debugger-prompt.png)
+
+Select your app's container group.
+
+![Screenshot of container group selection](./images/compose/select-container-group.png)
```

Comment:
Typo in the file name prevents the screenshot from being shown

Reply:
changed to standard image path format